styles/_mandara.scss

```
@@ -79,19 +79,32 @@ body:has(.mandara-editor) {
   display: grid;
   grid-template-columns: 1fr 400px;
   gap: var(--spacing-xl);
+  // 9マスグリッドを1画面に収めるため高さを制約
```

Stylelint (17.6.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)